restructuredtext_lint/sphinx.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import absolute_import
+
+import contextlib
+
+try:
+    import sphinx  # noqa
+
+    from sphinx.domains import c as c_domain
+    from sphinx.domains import cpp as cpp_domain
+    from sphinx.domains import javascript as javascript_domain
+    from sphinx.domains import python as python_domain
+    from sphinx.domains import std as std_domain
```

Comment:
Thats all that sphinx domains have ;)

  >>> import sphinx
  >>> from sphinx import domains
  >>> dir(domains)
  ['BUILTIN_DOMAINS', 'CDomain', 'CPPDomain', 'Domain', 'Index', 'JavaScriptDomain', 'ObjType', 'PythonDomain', 'ReSTDomain', 'SphinxError', 'StandardDomain', '_', '__builtins__', '__doc__', '__file__', '__name__', '__package__', '__path__', 'c', 'cpp', 'javascript', 'python', 'rst', 'std']